### PR TITLE
 Guide for complete data protection feature usage documentation

### PR DESCRIPTION
**Pull request by shobha2626**
1. **What this PR does / why we need it**:   Guide for complete data protection feature usage documentation

### CODE REVIEW

File: docs/data-sources/helm_feature.md [hunk 1]
```diff
@@ -0,0 +1,73 @@
+---
+# generated by https://github.com/hashicorp/terraform-plugin-docs
+page_title: "tanzu-mission-control_helm_feature Data Source - terraform-provider-tanzu-mission-control"
+subcategory: ""
+description: |-
+  
+---
+
+# tanzu-mission-control_helm_feature (Data Source)
+
```
Comment (ishangupta-ds) [inline docs/data-sources/helm_feature.md:10]: Please add content from official docs. 

You can add, this data source allows us to enable or disable the feature and ensure proper cleanup on disable.

File: docs/guides/tanzu-mission-control_data_protection.md [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+
+* Entire/Full Cluster
+* Selected Namespaces
+* Resources Selection By Label Selector
+
+For more information regarding scheduled backups, see [Scheduled Backups][backup-schedule].
+
+[backup-schedule]: https://docs.vmware.com/en/VMware-Tanzu-Mission-Control/services/tanzumc-using/GUID-89926F80-050A-4F1C-9D04-D56D5F453995.html?hWord=N4IghgNiBcIEZgMYGsCuAHABAZ0QCwFMATVCAkAXyA
+
+## Complete Data Protection Feature
```
Comment (vmw-vjn) [inline docs/guides/tanzu-mission-control_data_protection.md:41]: Suggest rephrasing the section

> Sample usage of Data Protection
You can use the following template as reference for enabling all stages of data protection feature  of Tanzu Mission Control using Terraform (i.e.) Enable data-protection for cluster, Set a Target Location of backup and finally, set a backup schedule for periodic data protection.

File: docs/data-sources/helm_feature.md (hunk 1 above)
Comment (shobha2626) [reply docs/data-sources/helm_feature.md]: It's a data source and we only read with the help of data source.

Will take up helm documentation gap in the next PR

File: docs/guides/tanzu-mission-control_data_protection.md (hunk 2 above)
Comment (shobha2626) [reply docs/guides/tanzu-mission-control_data_protection.md]: Done

**Review by vmw-vjn — APPROVED** [review]

**Review by ramya-bangera — APPROVED** [review]

**Pull request merged**